<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7R5CGPG" name="com.example.tests.ContactModificationTests" tests="1" failures="0" timestamp="14 Apr 2017 13:32:50 GMT" time="0.000" errors="1">
  <testcase name="modifySomeContact" time="0.000" classname="com.example.tests.ContactModificationTests">
    <error type="org.testng.TestNGException" message="
Some DataProvider public java.util.Iterator com.example.tests.TestBase.randomValidContactGenerator(java.lang.String) parameters unresolved:  at 1 type class java.lang.String
">
      <![CDATA[org.testng.TestNGException: 
Some DataProvider public java.util.Iterator com.example.tests.TestBase.randomValidContactGenerator(java.lang.String) parameters unresolved:  at 1 type class java.lang.String

	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:129)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1270)
	at org.testng.internal.Invoker.createParameters(Invoker.java:985)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1075)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </error>
  </testcase> <!-- modifySomeContact -->
</testsuite> <!-- com.example.tests.ContactModificationTests -->
